src/main/java/Duke.java

```diff
@@ -1,10 +1,189 @@
+import java.util.*;
+
 public class Duke {
+
+    static void level1() {
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?");
+
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+
+        while (!input.equals("bye")) {
+            System.out.println(input);
+            input = sc.nextLine();
+        }
+
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+
+    static void level2() {
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?");
+
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+
+        List<String> lst = new ArrayList<>();
```

Comment:
Perhaps the parameter name can be something like commands? lst may sound a bit unclear to readers.